pinot-controller/src/main/resources/app/pages/Query.tsx

```diff
@@ -47,6 +47,7 @@ import PinotMethodUtils from '../utils/PinotMethodUtils';
 import '../styles/styles.css';
 import {Resizable} from "re-resizable";
 import { useHistory, useLocation } from 'react-router';
+import sqlFormatter from '@sqltools/formatter';
```

Comment:
The last update to this package was around a year ago.

I had also found this one earlier which seems to be quite actively maintained: https://www.npmjs.com/package/sql-formatter

Should we explore sql-formatter instead?

Reply:
Hi @ankitsultana the sql-formatter is currently in [maintenance mode](https://github.com/sql-formatter-org/sql-formatter#the-future), no new features are being added. Also it's [incompatible](https://github.com/sql-formatter-org/sql-formatter#module-parse-failed-unexpected-token) with the babel configuration we are using. The above `sqltools/formatter` actively get's maintained and has very close parity to sql-formatter. It has new updates as well. Thanks!